src/plugins/keyboard-shortcuts.js

```diff
@@ -36,6 +37,7 @@ function keyboardShortcuts(app, opts, done){
   } = keypress;
 
   const {
+    showHelpOverlay,
```

Comment:
was trying to keep all the `show*Overlay` methods next to each other for easier refactoring